docs/docs/components.mdx

```diff
@@ -1474,6 +1474,18 @@ Intent classifiers assign one of the intents defined in the domain file to incom
     set `weight_sparsity` to 1 as this would result in all kernel weights being 0, i.e. the model is not able
     to learn.
 
+  * `constrain_similarities`:
```

Comment:
did we not describe this before anywhere? 🤔

Reply:
We did in the section where all configuration options of DIET are described. But this section contains options which we recommend users to tune. That's why I thought it would be good to have it here as well.